backend/src/index.ts

```diff
@@ -15,6 +16,7 @@ app.use(express.urlencoded({ extended: false }));
 app.use(cors());
 
 app.use('/api', router);
+app.use(GlobalErrorHandler.handleError);
```

Comment:
Global error handler that catches errors and sends back an internal_error (or other base errors) for anything that we haven't defined the handler to do